.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+  - "node"
```

Comment:
it runs `npm test` which runs both (see scripts in `package.json`). it's free for open source.